### ISSUE TITLE
Unused import does not consider rangeUntil usage

### ISSUE THREAD

**Issue by hudson155** [detekt/detekt#7103]
## Expected Behavior
Importing `rangeUntil` and using the `..<` operator should not report an issue for the unused import rule.

## Observed Behavior
It does report an issue.

## Steps to Reproduce
```kotlin
package highbeam.api.some

import highbeam.rest.endpoint.RestEndpoint
import highbeam.type.dateRange.ZonedDateTimeRange
import highbeam.type.dateRange.rangeUntil
import java.time.ZonedDateTime
import java.util.UUID

public object SomeApi {
  @RestEndpoint.Method("GET")
  @RestEndpoint.Path("/foo/bar/baz")
  public data class Search(
    @QueryParam val businessGuid: UUID,
    @QueryParam val dateRangeStart: ZonedDateTime,
    @QueryParam val dateRangeEndExclusive: ZonedDateTime,
  ) : RestEndpoint<Nothing>() {
    val dateRange: ZonedDateTimeRange = dateRangeStart..<dateRangeEndExclusive
  }
}
```

## Your Environment
* Version of detekt used: 1.23.6
* Version of Gradle used (if applicable): 8.6

## Other Context
This is not an issue with `rangeTo`.
